modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java

```diff
@@ -55,6 +62,34 @@ public CommerceOrderListDisplayContext(
 		_keywords = ParamUtil.getString(renderRequest, "keywords");
 	}
 
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		List<DropdownItem> bulkActions = new ArrayList<>();
+
+		PortletDisplay portletDisplay =
+			_commerceOrderRequestHelper.getPortletDisplay();
+
+		ThemeDisplay themeDisplay =
+			_commerceOrderRequestHelper.getThemeDisplay();
+
+		PortletURL portletURL = PortletURLFactoryUtil.create(
+			_commerceOrderRequestHelper.getRequest(), portletDisplay.getId(),
+			themeDisplay.getPlid(), PortletRequest.ACTION_PHASE);
+
+		portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
+		portletURL.setParameter(
+			ActionRequest.ACTION_NAME, "commerce_order/edit_commerce_order");
```

Comment:
@Commando18 missing the `/` in the front.
```suggestion
			ActionRequest.ACTION_NAME, "/commerce_order/edit_commerce_order");
```